Meta: Can no longer run under Windows using WSL/QEMU workflow

As introduced in ecccd511fac7662f9ab9bf499cbe0c2be49b2440 QEMU is run with the `-qmp` argument to enable the QMP protocol, useful for interacting with QEMU instances. In particular, it was recently used to help profile Serenity's kernel and find inefficiencies. However, Meta/run.sh was updated to always create a unix socket on start. Windows hosts cannot create a unix socket - while execution is started on Linux, the actual QEMU executable is from on the Windows host - hence no unix socket can be created.

### Reproduction
- Clone Serenity, and ensure the Windows build instructions are followed - in this test case, Ubuntu on WSL2 was used on a Windows 11 host.
- Execute Meta/serenity.sh run

### Result
```
[0/1] cd /home/nathan/git/serenity/Build/i686 && /usr/bin/cmake -E env SERENITY_ARCH=i686 /home/nathan/git/serenity/Meta/run.sh
qemu-system-i386.exe: -qmp unix:qmp-sock,server,nowait: unix sockets are not available on windows
FAILED: CMakeFiles/run
cd /home/nathan/git/serenity/Build/i686 && /usr/bin/cmake -E env SERENITY_ARCH=i686 /home/nathan/git/serenity/Meta/run.sh
ninja: build stopped: subcommand failed.
```

### Expected Output
QEMU should launch with an instance of SerenityOS running.

--- a/Meta/run.sh
+++ b/Meta/run.sh
@@ -213,9 +213,13 @@ if [ -z "$SERENITY_MACHINE" ]; then
         -device i82801b11-bridge,id=bridge3 -device sdhci-pci,bus=bridge3
         -device ich9-ahci,bus=bridge3
         -chardev stdio,id=stdout,mux=on
-        -qmp unix:qmp-sock,server,nowait
         "
     fi
+fi
+
+if [ "$NATIVE_WINDOWS_QEMU" -ne "1" ]; then
+    SERENITY_MACHINE="$SERENITY_MACHINE
+    -qmp unix:qmp-sock,server,nowait"
 fi
 
 

